Hymns/Common/BottomSheetView.swift

```diff
@@ -0,0 +1,43 @@
+import SwiftUI
+
+private enum Constants {
```

Comment:
The name "Constants" is a little generic. If you aren't really using this class, you can just remove it and just put the radius directly into the cornerRadius field

Reply:
Removed